rtloader/three/three.h

```diff
@@ -74,6 +74,7 @@ class Three : public RtLoader
 
     // const API
     py_info_t *getPyInfo();
+    void freePyInfo(py_info_t *);
```

Comment:
As mentioned, I believe this belongs in the base class - a few things do as well, but let's just not add more duplicate code if we can avoid it.